Makefile

```diff
@@ -60,4 +60,7 @@ distclean: clean
 	rm -rf .cache
 	rm -rf vendor
 
+functional:
+	cd functional && $(MAKE) all
```

Comment:
I think make has a flag for cwd so you could potentially collapse this down to a single command